bokehjs/src/lib/models/widgets/button_group.ts

```diff
@@ -39,7 +39,10 @@ export abstract class ButtonGroupView extends OrientedControlView {
         class: [buttons.btn, buttons[`btn_${this.model.button_type}` as const]],
         disabled: this.model.disabled,
       }, label)
-      button.addEventListener("click", () => this.change_active(i))
+      button.addEventListener("click", () => {
+        this.change_active(i)
+        this.model.trigger_event(new ButtonClick())
```

Comment:
We were ging to remove ButtonClick from stateful buttons? radio/checkbox group should use "on change" for both Python and JS